_targets.R

```diff
@@ -21,12 +24,18 @@ list(
   tar_target(oldest_active_sites, find_oldest_sites(states, parameter)),
 
   tar_map(
-    values = tibble(state_abb = states),
+    tibble(state_abb = states) %>%
+      mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
     tar_target(nwis_inventory, dplyr::filter(oldest_active_sites,
                                              state_cd == state_abb)),
-    tar_target(data, get_site_data(nwis_inventory, state_abb, parameter))
+    tar_target(data, get_site_data(nwis_inventory, state_abb, parameter)),
```

Comment:
I suggest adjusting this to be named with a bit more specifics, e.g. `nwis_data`, because the subsequent steps that call on just `data` are a bit less clear as you just read them.